kafka/deltaker-ingestor/src/main/kotlin/no/nav/amt/tiltak/kafka/deltaker_ingestor/DeltakerIngestorImpl.kt

```diff
@@ -64,36 +64,49 @@ class DeltakerIngestorImpl(
 				}
 			}
 
-		val status = DeltakerStatusInsert(
-			id = deltakerDto.status.id ?: UUID.randomUUID(),
-			deltakerId = deltakerDto.id,
-			type = deltakerDto.status.type,
-			aarsak = deltakerDto.status.aarsak,
-			aarsaksbeskrivelse = deltakerDto.status.aarsaksbeskrivelse,
-			gyldigFra = deltakerDto.status.gyldigFra,
-		)
+		val erKometDeltaker = unleashService.erKometMasterForTiltakstype(tiltakstype)
+		if (erKometDeltaker) {
+			val status = DeltakerStatusInsert(
+				id = deltakerDto.status.id ?: UUID.randomUUID(),
+				deltakerId = deltakerDto.id,
+				type = deltakerDto.status.type,
+				aarsak = deltakerDto.status.aarsak,
+				aarsaksbeskrivelse = deltakerDto.status.aarsaksbeskrivelse,
+				gyldigFra = deltakerDto.status.gyldigFra,
+			)
 
-		val deltakerUpsert = DeltakerUpsert(
-			id = deltakerDto.id,
-			statusInsert = status,
-			startDato = deltakerDto.oppstartsdato,
-			sluttDato = deltakerDto.sluttdato,
-			dagerPerUke = deltakerDto.dagerPerUke,
-			prosentStilling = deltakerDto.prosentStilling?.toFloat(),
-			registrertDato = deltakerDto.innsoktDato.atStartOfDay(),
-			gjennomforingId = gjennomforingId,
-			innsokBegrunnelse = deltakerDto.bestillingTekst,
-			innhold = deltakerDto.innhold,
-			kilde = Kilde.KOMET,
-			forsteVedtakFattet = deltakerDto.forsteVedtakFattet,
-			sistEndretAv = deltakerDto.sistEndretAv,
-			sistEndretAvEnhet = deltakerDto.sistEndretAvEnhet
-		)
+			val deltakerUpsert = DeltakerUpsert(
+				id = deltakerDto.id,
+				statusInsert = status,
+				startDato = deltakerDto.oppstartsdato,
+				sluttDato = deltakerDto.sluttdato,
+				dagerPerUke = deltakerDto.dagerPerUke,
+				prosentStilling = deltakerDto.prosentStilling?.toFloat(),
+				registrertDato = deltakerDto.innsoktDato.atStartOfDay(),
+				gjennomforingId = gjennomforingId,
+				innsokBegrunnelse = deltakerDto.bestillingTekst,
+				innhold = deltakerDto.innhold,
+				kilde = deltakerDto.kilde ?: Kilde.ARENA,
+				forsteVedtakFattet = deltakerDto.forsteVedtakFattet,
+				sistEndretAv = deltakerDto.sistEndretAv,
+				sistEndretAvEnhet = deltakerDto.sistEndretAvEnhet
+			)
+
+			transactionTemplate.executeWithoutResult {
+				deltakerService.upsertDeltaker(deltakerDto.personalia.personident, deltakerUpsert, erKometDeltaker)
```

Comment:
Går utifra at dette er ingestor for deltaker-v2..
Så da er denne ansvarlig for å publisere kometdeltakere til deltaker-v1.
Hvis det ikke er komet deltaker så publiseres også data tilbake til deltaker-v2 med potensielt kilde=arena isteden for null? Er dette for å rydde i dataene på topicen?